src/Resources/contao/dca/tl_list_config.php

```diff
@@ -759,7 +760,9 @@
     $dca['subpalettes']['addDetails'] = 'useModalExplanation,useModal,'.$dca['subpalettes']['addDetails'];
 }
 
-if (System::getContainer()->get('huh.utils.container')->isBundleActive('Terminal42\DcMultilingualBundle\Terminal42DcMultilingualBundle')) {
+if (in_array('Terminal42\DcMultilingualBundle\Terminal42DcMultilingualBundle',
```

Comment:
Können wir nicht das ganze DC_Multilingual-Zeug in einen Event-Listener packen, damit es nicht so verteilt ist?